TSOClient/tso.simantics/packages.config

```diff
@@ -1,4 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="MonoGame.Framework.WindowsGL" version="3.4.0.459" targetFramework="net4" />
+  <!--Although we don't do any network stuff, this gives us access to IoBuffer for serialization-->
+  <package id="Mina" version="2.0.11" targetFramework="net4" />
```

Comment:
Serialization of outfits, it just needs IoBuffer. we could always rip that part out into common.